picfixPy/contrast.py

```diff
@@ -16,5 +22,80 @@ def contrast(input_img, intensity=5, output_img):
     
     Example
     ============
-    contrast("../img/image.png", 2, "../img/contrast_image.png")
-    '''
\ No newline at end of file
+    contrast("../img/image.png", 5, display=False, "../img/contrast_image.png")
+    '''
+
+    # load input image
+    # lift pixel value restrictions between 0 and 255 temporarily
+    try: 
+        img = skimage.io.imread(input_img).astype(int)
+    
+    except AttributeError:
+        print("Please use a string to specify the file path for an input image.")
+        raise
+    
+    except OSError:
+        print("Please provide the path to an image file.")
+        raise
+    
+    except FileExistsError:
+        print("Cannot find image file.")
+            
+    except Exception as error:
```

Comment:
should we include an exception for image type?

Reply:
Nice catch!
The OSError is meant for catching illegal file type, so I rewrote the error message to be more clear!

The new error message now reads:
```
    except OSError:
        print("Please provide an image file.")
        raise
```